docs/guides/docs/intro-to-sway/prerequisites.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Prerequisites
+category: Intro to Sway
+parent:
+  label: Guides
+  link: /guides
+---
+
```

Comment:
Can you make sure each page has an h1 header (like this page could have `# Prerequisites`)